static/app/views/dashboards/widgets/timeSeriesWidget/useReleaseStats.tsx

```diff
@@ -8,11 +9,28 @@ interface ReleaseMetaBasic {
   version: string;
 }
 
+interface UseReleaseStatsParams {
+  datetime: Parameters<typeof normalizeDateTimeParams>[0];
+  environments: readonly string[];
+  projects: readonly number[];
```

Comment:
I decided to change this from using `PageFilters` because it was annoying to use this when you don't give it pagefilter selection params directly (e.g. issue details uses eventView). Instead, since we pass the datetime options to `normalizeDateTimeParams`, use its param type.